rules/xcodeproj.bzl

```diff
@@ -61,11 +68,15 @@ def _xcodeproj_aspect_impl(target, ctx):
                 direct_srcs = [],
             ),
         )
-        target_info = _TargetInfo(direct_target = info, targets = depset([info], transitive = _get_attr_values_for_name(deps, _TargetInfo, "targets")))
+        direct_targets = [info]
+        if test_host_target:
+            direct_targets.extend(test_host_target[_TargetInfo].direct_targets)
+        target_info = _TargetInfo(direct_targets = direct_targets, targets = depset([info], transitive = _get_attr_values_for_name(deps, _TargetInfo, "targets")))
```

Comment:
Even if the developer does not want to load transitive targets, we still have to load the test host application as a target. Otherwise, Xcode will complain (since the xcode test target is pointing to a test host application.)